### DISCUSSION TITLE
为什么我的be总是全体挂掉

### DISCUSSION THREAD

**Discussion by 12155797** [apache/doris#26657] [A - General / Q&A]
日志：start time: Thu May 13 15:47:25 CST 2021
tcmalloc: large alloc 1195728896 bytes == 0x64968000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1246912512 bytes == 0x64968000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0xb4964000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1811947520 bytes == 0xb4964000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0xb4964000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1949515776 bytes == 0x7fbdfc5b8000 @  0x269d2a3 0x280f95b 0x27c1209
start time: Mon Jul 11 09:02:15 CST 2022
tcmalloc: large alloc 1195728896 bytes == 0x3c801a000 @  0x269d393 0x280f95b 0x27c1209
tcmalloc: large alloc 1246912512 bytes == 0x4023ce000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x40a47e000 @  0x269d393 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x3baf78000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1811947520 bytes == 0x3baf78000 @  0x269d2a3 0x280f95b 0x27c1209
start time: Sun Oct 9 19:32:28 CST 2022
tcmalloc: large alloc 1195728896 bytes == 0x6ab636000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1246912512 bytes == 0x6a67fe000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x6a67fe000 @  0x269d393 0x280f95b 0x27c1209
tcmalloc: large alloc 1811947520 bytes == 0x6a67fe000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x6a67fe000 @  0x269d2a3 0x280f95b 0x27c1209
start time: Mon Oct 9 17:16:00 CST 2023
tcmalloc: large alloc 1195728896 bytes == 0x643660000 @  0x269d393 0x280f95b 0x27c1209
tcmalloc: large alloc 1246912512 bytes == 0x68aab6000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x643660000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x643660000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1811947520 bytes == 0x643660000 @  0x269d2a3 0x280f95b 0x27c1209
*** Aborted at 1698375448 (unix time) try "date -d @1698375448" if you are using GNU date ***
PC: @0x2760970 memcpy
*** SIGSEGV (@0x0) received by PID 109499 (TID 0x7f561c17d700) from PID 0; stack trace: ***
@0x32f360f520 (unknown)
@0x2760970 memcpy
@0x161274a doris::PartitionedHashTableCtx::EvalRow()
@0x15bddca doris::PartitionedAggregationNode::EvalAndHashPrefetchGroup<>()
@0x15bd658 doris::PartitionedAggregationNode::ProcessBatchStreaming()
@0x15b63a8 doris::PartitionedAggregationNode::GetRowsStreaming()
@0x15b9d76 doris::PartitionedAggregationNode::GetNextInternal()
@0x15b9f1f doris::PartitionedAggregationNode::get_next()
@0x10c2e6f doris::PlanFragmentExecutor::get_next_internal()
@0x10c4362 doris::PlanFragmentExecutor::open_internal()
@0x10c4c3f doris::PlanFragmentExecutor::open()
@0x103d9fe doris::FragmentExecState::execute()
@0x1040d96 doris::FragmentMgr::_exec_actual()
@0x104686c std::_Function_handler<>::_M_invoke()
@0x11b5892 doris::ThreadPool::dispatch_thread()
@0x11ad0bf doris::Thread::supervise_thread()
@0x32f3606d14 (unknown)
@0x32f32eac4d (unknown)
start time: Thu Nov 2 09:34:31 CST 2023
tcmalloc: large alloc 1195728896 bytes == 0x6008f8000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1246912512 bytes == 0x6008f8000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x64fdfe000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 1929256960 bytes == 0x6008f8000 @  0x269d393 0x280f95b 0x27c1209
tcmalloc: large alloc 1811947520 bytes == 0x6008f8000 @  0x269d2a3 0x280f95b 0x27c1209
tcmalloc: large alloc 8796093022208 bytes == (nil) @  0x269d393 0x281014b 0x281049a 0x10fe6bb 0x10ac8f4 0x102461b 0x1026405 0x161274a 0x15bddca 0
x15bd658 0x15b63a8 0x15b9d76 0x15b9f1f 0x10c2e6f 0x10c4362 0x10c4c3f 0x103d9fe 0x1040d96 0x104686c 0x11b5892 0x11ad0bf 0x32f3606d14
*** Aborted at 1699406077 (unix time) try "date -d @1699406077" if you are using GNU date ***
PC: @0x2760a40 memcpy
*** SIGSEGV (@0xc10) received by PID 77533 (TID 0x7f039796d700) from PID 3088; stack trace: ***
@0x32f360f520 (unknown)
@0x2760a40 memcpy
@0x161274a doris::PartitionedHashTableCtx::EvalRow()
@0x15bddca doris::PartitionedAggregationNode::EvalAndHashPrefetchGroup<>()
@0x15bd658 doris::PartitionedAggregationNode::ProcessBatchStreaming()
@0x15b63a8 doris::PartitionedAggregationNode::GetRowsStreaming()
@0x15b9d76 doris::PartitionedAggregationNode::GetNextInternal()
@0x15b9f1f doris::PartitionedAggregationNode::get_next()
@0x10c2e6f doris::PlanFragmentExecutor::get_next_internal()
@0x10c4362 doris::PlanFragmentExecutor::open_internal()
@0x10c4c3f doris::PlanFragmentExecutor::open()
@0x103d9fe doris::FragmentExecState::execute()
@0x1040d96 doris::FragmentMgr::_exec_actual()
@0x104686c std::_Function_handler<>::_M_invoke()
@0x11b5892 doris::ThreadPool::dispatch_thread()
@0x11ad0bf doris::Thread::supervise_thread()
@0x32f3606d14 (unknown)
@0x32f32eac4d (unknown)
start time: Wed Nov 8 10:28:05 CST 2023
tcmalloc: large alloc 4611686018427387904 bytes == (nil) @  0x269d2a3 0x281014b 0x281049a 0x10fe6bb 0x10ac8f4 0x102461b 0x1026405 0x161274a 0x15b
ddca 0x15bd658 0x15b63a8 0x15b9d76 0x15b9f1f 0x10c2e6f 0x10c4362 0x10c4c3f 0x103d9fe 0x1040d96 0x104686c 0x11b5892 0x11ad0bf 0x32f3606d14
*** Aborted at 1699498159 (unix time) try "date -d @1699498159" if you are using GNU date ***
PC: @0x2760af0 memcpy
*** SIGSEGV (@0x0) received by PID 95105 (TID 0x7f2d67d7c700) from PID 0; stack trace: ***
@0x32f360f520 (unknown)
@0x2760af0 memcpy
@0x161274a doris::PartitionedHashTableCtx::EvalRow()
@0x15bddca doris::PartitionedAggregationNode::EvalAndHashPrefetchGroup<>()
@0x15bd658 doris::PartitionedAggregationNode::ProcessBatchStreaming()
    @0x15b63a8 doris::PartitionedAggregationNode::GetRowsStreaming()
@0x15b9d76 doris::PartitionedAggregationNode::GetNextInternal()
@0x15b9f1f doris::PartitionedAggregationNode::get_next()
@0x10c2e6f doris::PlanFragmentExecutor::get_next_internal()
@0x10c4362 doris::PlanFragmentExecutor::open_internal()
@0x10c4c3f doris::PlanFragmentExecutor::open()
@0x103d9fe doris::FragmentExecState::execute()
@0x1040d96 doris::FragmentMgr::_exec_actual()
@0x104686c std::_Function_handler<>::_M_invoke()
@0x11b5892 doris::ThreadPool::dispatch_thread()
@0x11ad0bf doris::Thread::supervise_thread()
@0x32f3606d14 (unknown)
@0x32f32eac4d (unknown)
start time: Thu Nov 9 14:21:50 CST 2023
tcmalloc: large alloc 4611686018427387904 bytes == (nil) @  0x269d2a3 0x281014b 0x281049a 0x10fe6bb 0x10ac8f4 0x102461b 0x1026405 0x161274a 0x15b
ddca 0x15bd658 0x15b63a8 0x15b9d76 0x15b9f1f 0x10c2e6f 0x10c4362 0x10c4c3f 0x103d9fe 0x1040d96 0x104686c 0x11b5892 0x11ad0bf 0x32f3606d14
*** Aborted at 1699511007 (unix time) try "date -d @1699511007" if you are using GNU date ***
PC: @0x2760af0 memcpy
*** SIGSEGV (@0x41b) received by PID 38924 (TID 0x7f4d3497c700) from PID 1051; stack trace: ***
@0x32f360f520 (unknown)
@0x2760af0 memcpy
@0x161274a doris::PartitionedHashTableCtx::EvalRow()
@0x15bddca doris::PartitionedAggregationNode::EvalAndHashPrefetchGroup<>()
@0x15bd658 doris::PartitionedAggregationNode::ProcessBatchStreaming()
    @0x15b63a8 doris::PartitionedAggregationNode::GetRowsStreaming()
@0x15b9d76 doris::PartitionedAggregationNode::GetNextInternal()
@0x15b9f1f doris::PartitionedAggregationNode::get_next()
@0x10c2e6f doris::PlanFragmentExecutor::get_next_internal()
@0x10c4362 doris::PlanFragmentExecutor::open_internal()
@0x10c4c3f doris::PlanFragmentExecutor::open()
@0x103d9fe doris::FragmentExecState::execute()
@0x1040d96 doris::FragmentMgr::_exec_actual()
@0x104686c std::_Function_handler<>::_M_invoke()
@0x11b5892 doris::ThreadPool::dispatch_thread()
@0x11ad0bf doris::Thread::supervise_thread()
@0x32f3606d14 (unknown)
@0x32f32eac4d (unknown)

数据库查询：![image](https://github.com/apache/doris/assets/150329809/10120e05-4fcf-47fd-8817-d2d989fd7f47)

**Comment by 12155797** [comment]
刚启动完成，刷新之后又挂了，日志说三台Linux时间不对，我用date命令执行，时间是一致的

**Comment by 12155797** [comment]
求大佬告知

**Comment by LemonLiTree** [comment]
which doris version？

**Reply by 12155797** [reply]
是需要升级Doris的版本吗

**Reply by 12155797** [reply]
我这边看着是5.1.0的

**Reply by LemonLiTree** [reply]
`show backends;`
check version

**Reply by 12155797** [reply]
当前版本为0.13 大佬

**Reply by LemonLiTree** [reply]
The version is too low...

**Reply by 12155797** [reply]
更新到1.2.7.1可以吗大佬

**Reply by LemonLiTree** [reply]
0.13 -> 1.0 -> 1.1-> 1.2
